extensions/resource-metrics/resource-metrics-extention/ui/src/utils/api.ts

```diff
@@ -0,0 +1,51 @@
+export const apiCall = (url: string, headers: Record<string, string>) => {
+  return fetch(url, { headers })
+    .then((res) => res.json())
+    .then((res) => res)
+    .catch((err) => {
+      throw err;
+    });
+};
+
+export function getDashBoard({
```

Comment:
can you please add the short documentation on exported symbols?

Reply:
Added comments on top of each function